suites/reef/cephfs/tier-2_cephfs_test-metrics.yaml

```diff
@@ -151,4 +151,9 @@ tests:
       polarion-id: CEPH-83588303
       desc: Verify MDS client metrics by various operations and validate the metrics
       abort-on-fail: false
-
+  - test:
+      name: MDS client metrics scale testing by various operations
```

Comment:
Can we llimit the text mentioned under name:<>  ? This is because the same text used to form logfile name, if text length is more, the log file name would also be lengthier. 
We can add details under 'desc' as required.

Example:

 -
    test:
      abort-on-fail: false
      desc: "test cephfs nfs subvolume & subvolumegroup operations"
      module: cephfs_nfs.nfs_subvolume_subvolumegroup.py
      name: "cephfs nfs subvolume & subvolumegroup operations"

logfile : http://magna002.ceph.redhat.com/cephci-jenkins/results/openstack/RH/7.1/rhel-9/Regression/18.2.1-191/cephfs/128/tier-2_cephfs_test-nfs/cephfs_nfs_subvolume_&_subvolumegroup_operations_0.log

Reply:
reduce the number of characters in the desc and name